test/queryable_ordered_set_test.dart

```diff
@@ -230,6 +230,30 @@ void main() {
         expect(orderedSet.query<Bird>(), unorderedMatches(<Bird>[]));
         expect(orderedSet.toList(), isEmpty);
       });
+      test('#isRegistered', () {
+        final orderedSet = QueryableOrderedSet<Animal>(
+          Comparing.on((e) => e.name),
+        );
+        // No caches should be registered on a clean set
+        expect(orderedSet.isRegistered<Animal>(), false);
+        expect(orderedSet.isRegistered<Mammal>(), false);
+        orderedSet.register<Animal>();
+        // The Animal cache should report as registered after it has been
+        // registered
+        expect(orderedSet.isRegistered<Animal>(), true);
```

Comment:
expect(orderedSet.isRegistered<Mammal>(), false);

Reply:
but here it 2would test that registering Animal (a super class) doesn't trigger true to Mammal. in the end they are both true bc you register both by then

Reply:
Aaah, nooow I get what you wanted me to test. I guess you forgot the `<Mammal>` part in your comment?

Reply:
There we go, now it should hopefully be fine!
It is complaining about something weird in the dartdoc step though.

Reply:
AH markdown! so sorry hahaha